lib/providers/services/http_service.dart

```diff
@@ -251,18 +252,23 @@ class HttpService {
       return HttpMockResponse.invite;
     }
 
-    String inviteURL = "$baseURL/v1/chain/get_table_rows";
+    String reversedHash = reverseHash(inviteHash);
 
+    String inviteURL = "https://node.hypha.earth/v1/chain/get_table_rows";
     String request =
-        '{"json":true,"code":"join.seeds","scope":"join.seeds","table":"invites","table_key":"byhash","lower_bound":"$inviteHash","upper_bound":"$inviteHash","index_position":2,"key_type":"sha256","limit":100,"reverse":false,"show_payer":false}';
+        '{"json":true,"code":"join.seeds","scope":"join.seeds","table":"invites","lower_bound":"$reversedHash","upper_bound":"$reversedHash","index_position":2,"key_type":"sha256","limit":1,"reverse":false,"show_payer":false}';
     Map<String, String> headers = {"Content-type": "application/json"};
 
     Response res = await post(inviteURL, headers: headers, body: request);
 
     if (res.statusCode == 200) {
       Map<String, dynamic> body = jsonDecode(res.body);
 
-      InviteModel invite = InviteModel.fromJson(body);
+      InviteModel invite = InviteModel();
```

Comment:
It works similarly to all other methods in this class - it returns model with empty values in case when it's not found
When caller of this functions will be ready to catch errors properly - we can change this to throw errors I think

Reply:
Solved here: https://github.com/7flash/flutter_seeds_wallet/issues/18#event-3006772159

Reply:
So what exactly happens if there is an HTTP error - what happens in the user interface?

Reply:
My objection here is that the empty model can come from a few different cases - this is NOT OK.

Reply:
Just have different error return values - you don't need to throw an exception, but have an error return value, it's really standard, and we need to do it.

Reply:
Each time we have unclear error conditions, it adds entire days to our development... for finding and fixing bugs that we have no information about. 

Sentry does not solve this problem

Sentry does not make code better

Sentry cannot be used to make bad code into good code

Sentry can be used to help debug issues when the app is in production - but we will only ship good code to production and then it can help.

Reply:
The main thing is there is a clear distinction between

- the request was successful
- the request was successful and there was nothing found (empty list, 0, etc)
- the request was successful but with an unexpected result (invite not found when we expected an invite to be found)
- the request failed altogether (network error, server down.....)